src/free-style.ts

```diff
@@ -170,10 +170,11 @@ function stringifyProperties (properties: Properties) {
 
   for (const [name, value] of properties) {
     if (value != null) {
-      if (Array.isArray(value)) {
-        result.push(value.filter(x => x != null).map(x => styleToString(name, x)).join(';'))
+      const value2 = typeof value === 'function' ? value() : value
```

Comment:
No worries. I thought it was a nice solution too, I'd just rather maintain compatibility with the larger ecosystem. There's also possibilities in the future where we could inline all of `free-style`, but function calls make that a little trickier.